hollow/src/main/java/com/netflix/hollow/core/schema/HollowSchema.java

```diff
@@ -49,6 +49,9 @@
     private final String name;
 
     public HollowSchema(String name) {
+        if (name == null || name == "") {
```

Comment:
I suppose the test passes because `""` happens to be interned, but might be safer to use `.equals` anyway?

Reply:
Derp. Thanks!